public/modules/custom/grants_club_section/src/Validator/FieldValueValidator.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace Drupal\grants_club_section\Validator;
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a validator function for other cost/income elements.
+ */
+class FieldValueValidator {
+
+  /**
+   * Form element validation for club section adult age fields.
+   *
+   * Note that #required is validated by _form_validate() already.
+   */
+  public static function validateAdults(&$element, FormStateInterface $form_state) {
+    $parents = $element['#parents'];
+    $field = array_pop($parents);
+    $parent = $form_state->getValue($parents);
+    $tOpts = ['context' => 'grants_club_section'];
+
+    if (empty($parent['men'])
+      && empty($parent['women'])
+      && empty($parent['adultOthers'])
+      && empty($parent['seniorMen'])
+      && empty($parent['seniorWomen'])
+      && empty($parent['seniorOthers'])
+      && empty($parent['boys'])
+      && empty($parent['girls'])
+      && empty($parent['juniorOthers'])) {
+      $form_state->setError(
+        $element,
+        t(
+          "At least one age group is required.",
+          [],
+          $tOpts
+        )
+      );
+    }
+
+  }
+
+  /**
+   * Form element validation for club section adult age fields.
+   *
+   * Note that #required is validated by _form_validate() already.
+   */
+  public static function validateAdultHours(&$element, FormStateInterface $form_state) {
+    $parents = $element['#parents'];
+    $field = array_pop($parents);
+    $parent = $form_state->getValue($parents);
+    $tOpts = ['context' => 'grants_club_section'];
+
+    if (!empty($parent['men']) || !empty($parent['women']) || !empty($parent['adultOthers'])) {
```

Comment:
Eiks tää oo vähän hassusti, jos joku ei oo tyhjä, niin lyödään virhe 😅 ? 

![image](https://github.com/City-of-Helsinki/hel-fi-drupal-grants/assets/43133397/3ff00358-67be-40cd-83fb-9cabd53d620c)